### DISCUSSION TITLE
Error: Rowy API fail /firebaseToken?projectId=techhub-ui-2df643b4: 403

### DISCUSSION THREAD

**Discussion by zitonamurotelus** [AntlerVC/firetable#832] [Support / Q&A] [answered]
**Describe the bug**
After completing create project Instead of opening the project I see a 403 screen

**To Reproduce**
1.  Tried to add company DB to Rowy via create project
2. All the steps went well
3. Fails with 403 error when I press create project

**Expected behavior**
For the project to start

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 105.0.5195.102

**Comment by jackdund** [comment, marked as answer]
I had the same issue

**Reply by notsidney** [reply]
Hi everyone, this is fixed now.
